### PR TITLE
O3-1268: Default Identifiers types should not be required

### PR DESCRIPTION
**Pull request by vasharma05**
## Requirements

- [x] This PR has a title that briefly describes the work done, including the ticket number if there is a ticket.
- [x] My work conforms to the [**OpenMRS 3.0 Styleguide**](https://om.rs/styleguide).
- [x] I checked for feature overlap with [**existing widgets**](https://om.rs/directory).

## Summary

Default patient identifier types should not be required to be filled when registering a patient.

## Screenshots

*None.*

## Related Issue

https://issues.openmrs.org/browse/O3-1268").

## Other

*None.*

### CODE REVIEW

**Review by ZacButko — CHANGES_REQUESTED** [review]
Logic looks ok. Please restore translations if they are still in use.

File: packages/esm-outpatient-app/translations/en.json [hunk 1]
```diff
@@ -36,7 +36,6 @@
   "fields": "of the following fields",
   "firstName": "First name",
   "firstNameSort": "First name (a-z)",
-  "futureScheduledVisits": " {count, plural, one{{count} visit} other{{count} visits}} scheduled for dates in the future",
```
[the remark this answers is not in the document]
Comment (vasharma05) [reply packages/esm-outpatient-app/translations/en.json:39]: This translation is removed from the outpatient app.

File: packages/esm-patient-registration-app/translations/en.json [hunk 2]
```diff
@@ -4,8 +4,6 @@
   "addressHeader": "Address",
   "allFieldsRequiredText": "All fields are required unless marked optional",
   "autoGeneratedPlaceholderText": "Auto-generated",
-  "birthdayNotInTheFuture": "Birthdate cannot be in the future",
-  "birthdayRequired": "Birthdate is required",
```
[the remark this answers is not in the document]
Comment (vasharma05) [reply packages/esm-patient-registration-app/translations/en.json:8]: These translations removal will not occur again after #242 is merged, meanwhile restoring the translations back

**Comment by ZacButko** [comment]
OBE by PR https://github.com/openmrs/openmrs-esm-patient-management/pull/243

**Pull request closed without merging**